src/main/java/it/gov/pagopa/paymentupdater/consumer/MessageKafkaConsumer.java

```diff
@@ -31,24 +32,26 @@ public class MessageKafkaConsumer {
 	private String payload = null;
 
 	@KafkaListener(topics = "${kafka.message}", groupId = "consumer-message", containerFactory = "kafkaListenerContainerFactory", autoStartup = "${message.auto.start}")
-	public void messageKafkaListener(Payment paymentMessage)
-			throws JsonProcessingException, InterruptedException, ExecutionException {
+	public void messageKafkaListener(Payment paymentMessage) throws JsonProcessingException, InterruptedException, ExecutionException {
 		log.debug("Processing messageId=" + paymentMessage.getId() + " time=" + new Date().toString()
-				+ "paymentMessageContentType="
-				+ paymentMessage.getContent_type());
-		if (Objects.nonNull(paymentMessage.getContent_type())
-				&& paymentMessage.getContent_type().equals(MessageContentType.PAYMENT)) {
+				+ " paymentMessageContentType= " + paymentMessage.getContent_type());
+		if (Objects.nonNull(paymentMessage.getContent_type()) && paymentMessage.getContent_type().equals(MessageContentType.PAYMENT)) {
 			log.debug("Received message with id: {} ", paymentMessage.getId());
 			checkNullInMessage(paymentMessage);
 			payload = paymentMessage.toString();
-			var pp = paymentService.getPaymentByNoticeNumberAndFiscalCode(
-					paymentMessage.getContent_paymentData_noticeNumber(),
-					paymentMessage.getContent_paymentData_payeeFiscalCode());
+			var pp = paymentService.getPaymentByNoticeNumberAndFiscalCode(paymentMessage.getContent_paymentData_noticeNumber(),	paymentMessage.getContent_paymentData_payeeFiscalCode());
 			if (pp.isEmpty()) {
 				String rptId = paymentMessage.getContent_paymentData_payeeFiscalCode()
-						.concat(paymentMessage.getContent_paymentData_noticeNumber());
+                        .concat(paymentMessage.getContent_paymentData_noticeNumber());
 				paymentMessage.setRptId(rptId);
-				paymentServiceImpl.checkPayment(paymentMessage);
+				Map<String, String> map = paymentServiceImpl.checkPayment(paymentMessage);
```

Comment:
Can we consider to use a DTO instead of a Map?

Reply:
This has been fixed in the develop-rptid branch that will be merged later